crates/binstalk-fetchers/src/quickinstall.rs

```diff
@@ -280,10 +321,19 @@ impl QuickInstall {
             return Ok(());
         }
 
-        let url = self.stats_url.clone();
+        let mut url = self.stats_url.clone();
+        url.query_pairs_mut()
+            .append_pair("crate", &self.data.name)
+            .append_pair("version", &self.data.version)
+            .append_pair("target", &self.target_data.target)
+            .append_pair(
+                "agent",
+                concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION")),
+            )
```

Comment:
This can be encoded in the global `QUICK_INSTALL_STATS_URL`

Reply:
that might require urlencoding CARGO_PKG_VERSION (which is semver so might contain a +)